### ISSUE TITLE
Feature Request: Advanced Tuning System-UI Colors

### ISSUE THREAD

**Issue by FreezyExp** [GravityBox/GravityBox#2119]
TLDR version: 
Add systemui color overrides to advanced tuning (for example Oneplus's "clock_ten_digit_red")

Longer version:
I assume there is a design to not do device specific settings, which is why I'm formulating it very generically.
Either exposing any setting that contains a value string starting with # or all settings from colors.xml

This fits well in the design of Advanced Tuning and still limits access to semi-safe UI things.

Here is my full use-case:
```
I would love to nuke the red 1 tinting from the oneplus 7T lockscreen clock.
Ever since android 11 update, this has been annoying me, it is ugly and nothing can be done about it.
(they removed all lockscreen related options)

Using an overlay in substratum (no red in oneplus clock) does not seem to work with EdXposed (using a whitelist) & GravityBox installed.

I've collected some of the suspects I would like to adjust in res/values/colors.xml

<color name="clock_ten_digit_red">#ffeb0028</color>
<color name="oneplus_red_color">#ffeb0028</color>
